test/html-to-react-tests.ts

```diff
@@ -0,0 +1,681 @@
+import { equal, strictEqual, deepStrictEqual, ok } from "assert";
```

Comment:
Why are you diverging from the project's style of using single quotes? Could you revert to single quotes instead, so we avoid this intrusive change?